src/code/z_lifemeter.c

```diff
@@ -489,42 +495,42 @@ void HealthMeter_Draw(GlobalContext* globalCtx) {
     CLOSE_DISPS(gfxCtx, "../z_lifemeter.c", 606);
 }
 
-void HealthMeter_HandleCriticalAlarm(GlobalContext* globalCtx) {
+void HealthMeter_HandleBeatingHeart(GlobalContext* globalCtx) {
```

Comment:
(I know "handle" comes from current master)

I feel like `Update` instead of `Handle` makes more sense, since it's part of `Interface_Update` and indeed updates stuff
It's also not uncommon for functions dubbed "update" to play sound
```suggestion
void HealthMeter_UpdateBeatingHeart(GlobalContext* globalCtx) {
```